src/HackUnlock/toolbox.js

```diff
@@ -27,6 +26,7 @@ var HUToolbox = GObject.registerClass(class HUToolbox extends Toolbox {
 
     bindWindow(win) {
         win.get_style_context().add_class('HackUnlock');
+        win.getMainLockscreen().locked = false;
```

Comment:
Hmm... Previously this had a `false` unlock state, which corresponds to a `true` locked state, no?

Reply:
i will need to confirm with @llopis , but I think this one is never locked.. but if it is locked, then the quest for the Unlock/Welcome will need to provide a key, and we will need to use it here.